src/utils.tsx

```diff
@@ -1,4 +1,4 @@
-import { kebabCase } from "lodash";
+import kebabCase from "lodash.kebabcase";
 import React from "react";
```

Comment:
`src/utils.tsx` might be confusing since it conflicts with the `src/utils` directory. It might be better to move it as a separate PR though. I'll create an issue: https://github.com/Faire/mjml-react/issues/18